.travis/publish_docs.sh

```diff
@@ -19,9 +21,20 @@ BWHITE='\e[1;37m'       # Bold White
 
 function display_usage() {
   printf "${GREEN}Usage:\n"
-  printf "${YELLOW}      ./prepare_site.sh [version|dev]\n\n"
-  printf "${GREEN}   Example: ${YELLOW}./prepare_site.sh 2.2 ${GREEN}- Prepare docs for 2.2 release.\n"
-  printf "${GREEN}   Example: ${YELLOW}./prepare_site.sh dev ${GREEN}- Prepare docs for current SNAPSHOT.\n$RESET"
+  printf "${YELLOW}      ./publish_docs.sh [version|dev]\n\n"
```

Comment:
calling it `compile_docs.sh` makes it even better IMO, as we are only prepping the docs to be published. As Github publishes these docs automatically from gh-pages branch it really doesn't make sense to call it publish.